CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# 2.9.0
+* Add `filename` option in `produce_or_load`, overriding the default filename generated by `savename`
```

Comment:
```suggestion
* Add `filename` option in `produce_or_load`, possibly overriding the default filename generated by `savename`
```
The way it was phrased before it may have been misinterpreted as that we are actually changing the default behavior.

Reply:
Sounds good!